Plugin error in IntelliJ with a non-Flutter project

## What happened

I'm working with a Spring Boot project, but for some reason the Flutter plugin is thinking that this is a Flutter project and it's throwing the following error.

## Version information

IntelliJ IDEA `2019.2.3` • Flutter plugin `io.flutter 40.0.4` • Dart plugin `192.6817.14`

No Flutter sdk configured.

## Exception

null

```
java.lang.NoClassDefFoundError: com/android/tools/idea/gradle/project/sync/GradleSyncState
	at io.flutter.utils.AndroidUtils$1.condition(AndroidUtils.java:309)
	at com.intellij.util.WaitFor$1$1.condition(WaitFor.java:68)
	at com.intellij.util.WaitFor.<init>(WaitFor.java:51)
	at com.intellij.util.WaitFor.<init>(WaitFor.java:41)
	at com.intellij.util.WaitFor$1$1.<init>(WaitFor.java:65)
	at com.intellij.util.WaitFor$1.run(WaitFor.java:65)
Caused by: java.lang.ClassNotFoundException: com.android.tools.idea.gradle.project.sync.GradleSyncState PluginClassLoader[io.flutter, 40.0.4] com.intellij.ide.plugins.cl.PluginClassLoader@3e00b3fe
	at com.intellij.ide.plugins.cl.PluginClassLoader.loadClass(PluginClassLoader.java:75)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	... 6 more
```

--- a/resources/META-INF/plugin.xml
+++ b/resources/META-INF/plugin.xml
@@ -991,7 +991,6 @@
   </extensions>
 
   <extensions defaultExtensionNs="com.intellij">
-    <externalSystemTaskNotificationListener implementation="io.flutter.utils.FlutterExternalSystemTaskNotificationListener"/>
     <postStartupActivity implementation="io.flutter.ProjectOpenActivity"/>
     <postStartupActivity implementation="io.flutter.FlutterInitializer"/>
     <projectService serviceInterface="io.flutter.run.daemon.DeviceService"

--- a/resources/META-INF/plugin_template.xml
+++ b/resources/META-INF/plugin_template.xml
@@ -219,7 +219,6 @@
   </extensions>
 
   <extensions defaultExtensionNs="com.intellij">
-    <externalSystemTaskNotificationListener implementation="io.flutter.utils.FlutterExternalSystemTaskNotificationListener"/>
     <postStartupActivity implementation="io.flutter.ProjectOpenActivity"/>
     <postStartupActivity implementation="io.flutter.FlutterInitializer"/>
     <projectService serviceInterface="io.flutter.run.daemon.DeviceService"

--- a/resources/META-INF/studio-contribs.xml
+++ b/resources/META-INF/studio-contribs.xml
@@ -17,6 +17,7 @@
   </extensions>
 
   <extensions defaultExtensionNs="com.intellij">
+    <externalSystemTaskNotificationListener implementation="io.flutter.utils.FlutterExternalSystemTaskNotificationListener"/>
     <androidStudioInitializer implementation="io.flutter.FlutterStudioInitializer"/>
     <postStartupActivity implementation="io.flutter.FlutterStudioStartupActivity"/>
     <projectOpenProcessor implementation="io.flutter.project.FlutterStudioProjectOpenProcessor" order="after flutter"/>

--- a/resources/META-INF/studio-contribs_template.xml
+++ b/resources/META-INF/studio-contribs_template.xml
@@ -15,6 +15,7 @@
   </extensions>
 
   <extensions defaultExtensionNs="com.intellij">
+    <externalSystemTaskNotificationListener implementation="io.flutter.utils.FlutterExternalSystemTaskNotificationListener"/>
     <androidStudioInitializer implementation="io.flutter.FlutterStudioInitializer"/>
     <postStartupActivity implementation="io.flutter.FlutterStudioStartupActivity"/>
     <projectOpenProcessor implementation="io.flutter.project.FlutterStudioProjectOpenProcessor" order="after flutter"/>

